jib-maven-plugin/src/integration-test/java/com/google/cloud/tools/jib/maven/BuildDockerMojoIntegrationTest.java

```diff
@@ -99,12 +101,28 @@ public void testExecute_simple()
     Assert.assertEquals(
         "Hello, world. An argument.\n1970-01-01T00:00:01Z\nrw-r--r--\nrw-r--r--\nfoo\ncat\n"
             + "1970-01-01T00:00:01Z\n1970-01-01T00:00:01Z\n",
-        buildToDockerDaemonAndRun(simpleTestProject.getProjectRoot(), targetImage));
+        buildToDockerDaemonAndRun(simpleTestProject, targetImage));
     Assert.assertEquals(
         "1970-01-01T00:00:00Z",
         new Command("docker", "inspect", "-f", "{{.Created}}", targetImage).run().trim());
   }
 
+  @Test
+  public void testExecute_simple_extraDirectoriesFiltering()
+      throws DigestException, IOException, InterruptedException, VerificationException {
+    String targetImage = "simpleimage:maven" + System.nanoTime();
+    buildToDockerDaemon(simpleTestProject, targetImage, "pom-extra-dirs-filtering.xml");
+    String output =
+        new Command("docker", "run", "--rm", "--entrypoint=ls", targetImage, "-1R", "/extras")
+            .run();
+    // No "bar" or "*.txt" files. Specifically,
```

Comment:
how about something like this? 
```suggestion

    // No "bar" or "*.txt" files. Only copies the following:
```